api/envoy/api/v2/eds.proto

```diff
@@ -43,9 +43,12 @@ service EndpointDiscoveryService {
 //
 // With EDS, each cluster is treated independently from a LB perspective, with
 // LB taking place between the Localities within a cluster and at a finer
-// granularity between the hosts within a locality. For a given cluster, the
-// effective weight of a host is its load_balancing_weight multiplied by the
-// load_balancing_weight of its Locality.
+// granularity between the hosts within a locality. The load balancing weight
+// for an endpoint is divided by the sum of the weights of all endpoints in the
+// endpoint’s locality to produce a percentage of traffic for the endpoint.
+// This percentage is then further weighted by the endpoint’s locality’s load
```

Comment:
I think it would be good to be very explicit that the weighting works the same for the locality selection as for "endpoint in locality selection" instead of just saying "further weighted" - does that make sense?